docs/Tutorial_Time_Series_Snippets.ipynb

```diff
@@ -1,8 +1,51 @@
 {
```

Comment:
<ul><li>You said "common question" but you've only presented a statement. </li><li>Please remove "large" in "large time series" and then combine that sentence with the one following it</li><li>Please replace "his sleep" with "their sleep"</li></ul><p> </p><p>This paragraph seems somewhat forced and out of place:</p><p> </p><blockquote>For instance, the respiration data of a person recorded during his sleep can be reported by showing only the top-2 snippets. As an another example, snippets can be used to find different stages in the walking-nordic walking-skipping-running cycle of a person. Particularly, the y-axis acceleration data recorded by a chest-worn sensor can be summarized into top-4 snippets where each snippet shows a different type of exercise.  </blockquote><p> </p><p>This may be something that we'll need to think through and provide proper motivation without getting into too much unnecessary detail. </p><p> </p><p>I think that the <code>It is worthwhile to note what snippet is NOT </code>  part should not be in the starting section and should be its own section header</p><p> </p><p>This sentence isn't so clear:</p><p> </p><blockquote>one should specify the beginning and ending of subsequences and then apply clustering on the set of subseuqences</blockquote>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/397/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>